android/assets/jsons/Civ V - Gods & Kings/Buildings.json

```diff
@@ -1049,6 +1049,12 @@
 		"requiredTech": "Ecology",
 		"uniques": ["Must be next to [Desert]", "Cannot be built with [Nuclear Plant]"]
 	},
+	{
+		"name": "Recycling Center",
+		"maintenance": 3,
+		"requiredTech": "Ecology",
+		"uniques": ["Provides [2] [Aluminum]","Can have [5] of this building in your empire at most"]
```

Comment:
Change to "Limited to [] per Civilization" so we can reuse this for units, improvements etc